.github/workflows/sonar.yml

```diff
@@ -0,0 +1,61 @@
+name: SonarCloud Analysis
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, synchronize, reopened]
+  workflow_dispatch:
+  schedule:
+    - cron: '00 6 * * *' # 06:00 UTC every day
+
+jobs:
+  sonar:
+    if: github.repository == 'GoogleContainerTools/jib-extensions' # Only run on upstream branch
+    name: Build with Sonar
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Get current date
+        id: date
+        run: echo "::set-output name=date::$(date +'%Y-%m-%d' --utc)"
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Cache SonarCloud packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.sonar/cache
+          key: ${{ runner.os }}-sonar-${{ steps.date.outputs.date }}
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.m2/repository
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+      - name: Test w/ coverage
+        id: install1
```

Comment:
`id` probably unnecessary. And I wonder why the name is "install", while the step name is "test w/ coverage"? What's being installed and why? And I guess coverage info is an input to this Sonar Cloud?

Reply:
Sorry, that was a typo. Removed `id`. And yes, the coverage report from jacoco is an input to Sonar Cloud.